roles/ceph-common/tasks/main.yml

```diff
@@ -98,6 +98,9 @@
     - ceph_current_fsid.rc == 0
     - mon_group_name in group_names
 
+- name: include create_ceph_user_and_group.yml
+  include: create_ceph_user_and_group.yml
```

Comment:
either :
`include: ./installs/create_ceph_user_and_group.yml` 
or move `./installs/create_ceph_user_and_group.yml` to `roles/ceph-common/tasks`

Reply:
Yes, of course. Copy & paste issue..

Reply:
Move the file.

Reply:
From what I'm seeing in #2250 I'm not sure this PR makes sense

Reply:
You're right.